pybossa/api/__init__.py

```diff
@@ -376,6 +379,57 @@ def get_disqus_sso_api():
         return error.format_exception(e, target='DISQUS_SSO', action='GET')
 
 
+@jsonpify
+@csrf.exempt
```

Comment:
why exempt?

Reply:
This appears to be required for the ajax call to succeed from the client.

Reply:
You should pass a CSRF token with your POST request

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/400/commits/0db33773e60b99354a9fdea652b40bbe1330de8e